docs/wiki/deployment/aws-logging.md

```diff
@@ -1,26 +1,26 @@
-As of version 1.7.4, osquery can log results directly to Amazon AWS [Kinesis Streams](https://aws.amazon.com/kinesis/streams/) and [Kinesis Firehose](https://aws.amazon.com/kinesis/firehose/). For users of these services, `osqueryd` can eliminate the need for a separate log forwarding daemon running in your deployments.
+As of osquery version 1.7.4, osquery can log results directly to Amazon AWS [Kinesis Streams](https://aws.amazon.com/kinesis/streams/) and [Kinesis Firehose](https://aws.amazon.com/kinesis/firehose/). For users of these services, `osqueryd` can eliminate the need for a separate log forwarding daemon running in your deployments.
 
 ## Configuration
 
 The Kinesis Streams and Kinesis Firehose logger plugins are named `aws_kinesis` and `aws_firehose` respectively. They can be enabled as with other logger plugins using the config flag `logger_plugin`.
 
 Some configuration is shared between the two plugins:
 
-```
---aws_access_key_id VALUE               AWS access key ID override
---aws_profile_name VALUE                AWS config profile to use for auth and region config
---aws_region VALUE                      AWS region override
---aws_secret_access_key VALUE           AWS secret access key override
---aws_sts_arn_role VALUE                AWS STS assume role ARN
---aws_sts_region VALUE                  AWS STS assume role region
---aws_sts_session_name VALUE            AWS STS session name
---aws_sts_timeout VALUE                 AWS STS temporary credential timeout period in seconds (900-3600)
---aws_enable_proxy VALUE                Enable proxying of HTTP/HTTPS requests in AWS client config (true or false)
---aws_proxy_scheme VALUE                Proxy HTTP scheme for use in AWS client config (http or https)
---aws_proxy_host VALUE                  Proxy host for use in AWS client config
---aws_proxy_port VALUE                  Proxy port for use in AWS client config
---aws_proxy_username VALUE              Proxy username for use in AWS client config
---aws_proxy_password VALUE              Proxy password for use in AWS client config
+```shell
```

Comment:
This isn't shell code, and the addition of the `#` feels off. It's a cut&paste from a usage field. OTOH, I don't feel that stringly